tests/unit/helpers/emberTest.js

```diff
@@ -0,0 +1,30 @@
+/*eslint-env node, mocha, es6 */
+'use strict';
+// var rewire = require('rewire');
+// var ember = rewire('../../../lib/helpers/ember.js');
+var ember = require('../../../lib/helpers/ember');
+var expect = require('chai').expect;
+
+describe('ember helper', function () {
+  // before(function () {
+  // });
+
+  it('returns a value', function () {
+    expect(ember).to.be.ok();
+  });
+
+  it('returns local by default on linux', function () {
+    if (process.platform !== 'win32') {
```

Comment:
OK, found out that on windows, npm will create a `.bin\` directory at the top level inside `node_modules\` which contains the `.CMD` executables.  And it seems like `npm-which` will indeed do the trick! :-D